app/helpers/importmap/importmap_tags_helper.rb

```diff
@@ -24,7 +32,7 @@ def javascript_import_module_tag(*module_names)
   # Link tags for preloading all modules marked as preload: true in the `importmap`
   # (defaults to Rails.application.importmap), such that they'll be fetched
   # in advance by browsers supporting this link type (https://caniuse.com/?search=modulepreload).
-  def javascript_importmap_module_preload_tags(importmap = Rails.application.importmap)
+  def javascript_importmap_module_preload_tags(importmap)
```

Comment:
Can you explain why you removed this default value declaration?

Reply:
Same reason as with `javascript_inline_importmap_tag`, there is no single `Rails.application.importmap` anymore. :)